apps/wallet/src/ui/app/shared/forms/SelectField.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ChevronDown12 } from '@mysten/icons';
```

Comment:
it might be nice to follow the pattern I was trying to establish where the raw component that's independent of any form library goes in `/forms/controls` and the form library dependent field component goes in `/forms`